plugins/plugin-sass/README.md

```diff
@@ -0,0 +1,23 @@
+# @snowpack/plugin-sass
+
+This plugin adds [Sass](https://sass-lang.com/) support to any Snowpack project. With this plugin, you can import any  `*.scss` Sass from JavaScript and have it compile to CSS.
```

Comment:
And `*.sass?` (I don’t use it, but it’s still popular in some circles. Inb4 you get a request to support it.)